CacheMoney/frontend/src/components/NavBar.js

```diff
@@ -1,5 +1,9 @@
 import React from "react";
 
+// 		(From Gideon) it turns out that using "<a href="#">" just links to the top of the current page, and shouldn't
+// be vulnerable to URL injection (not 100% sure on that, though). But this does mean that this file shouldn't need any
+// immediete patches.
+
```

Comment:
This kind of comment should probably be in your team's discord channel instead of in the code